src/wallet/wallet2.cpp

```diff
@@ -4078,13 +4049,13 @@ boost::optional<wallet2::keys_file_data> wallet2::get_keys_file_data(const epee:
     json.AddMember("original_view_secret_key", value, json.GetAllocator());
   }
   
-  value2.SetInt(m_persistent_rpc_client_id ? 1 : 0);
+  value2.SetInt(0);
```

Comment:
Maybe comment in some way that the 3 RPC payment related values do not exist anymore / are not active anymore and are only written to make sure we don't have any compatibility problems?